test/parallel/test-module-relative-lookup.js

```diff
@@ -1,15 +1,23 @@
 'use strict';
 
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const _module = require('module'); // avoid collision with global.module
-const lookupResults = _module._resolveLookupPaths('./lodash');
-let paths = lookupResults[1];
 
-// Current directory gets highest priority for local modules
-assert.strictEqual(paths[0], '.');
+function runTest(moduleName) {
+  const lookupResults = _module._resolveLookupPaths(moduleName);
+  let paths = lookupResults[1];
 
-paths = _module._resolveLookupPaths('./lodash', null, true);
+  // Current directory gets highest priority for local modules
+  assert.strictEqual(paths[0], '.');
 
-// Current directory gets highest priority for local modules
-assert.strictEqual(paths && paths[0], '.');
+  paths = _module._resolveLookupPaths(moduleName, null, true);
+
+  // Current directory gets highest priority for local modules
+  assert.strictEqual(paths && paths[0], '.');
+}
+
+runTest('./lodash');
+if (common.isWindows) {
+  runTest('.\\lodash');
+}
```

Comment:
Should there be an `else` clause added that asserts that `runTest('.\\lodash')` on non-Windows throws?